capstone/client/src/components/ListPage.js

```diff
@@ -194,12 +206,20 @@ class ListPage extends Component {
       },
     ).then((res) => {
       this.setState({
+        listStatusMessage: "Your list has been saved!",
+        listSaveStatus: {
+          display: true,
+        },
         errorMessage: null,
         successMessage: "Your list has been saved!",
```

Comment:
Can we reuse the successMessage and the errorMessage below instead of creating a new field?